_data/tool_and_resource_list.yml

```
@@ -2634,3 +2634,11 @@
   registry:
     biotools: hpo
   url: https://hpo.jax.org/app/
+- description: free and open-source reference management software to manage bibliographic data and related research materials
```

```suggestion
- description: free and open-source reference management software to manage bibliographic data and related research materials
```
Add a capital for "free" and a full stop at the end? So: "Free and open-source reference management software to manage bibliographic data and related research materials."